apps/dashboard/services/graphql/queries/get-transactions.ts

```diff
@@ -0,0 +1,202 @@
+import { gql } from "@apollo/client";
+
+import { apollo } from "..";
+import {
+  GetFirstTransactionsDocument,
+  GetFirstTransactionsQuery,
+  GetPaginatedTransactionsDocument,
+  GetPaginatedTransactionsQuery,
+} from "../generated";
+
+// TODO remove not used fields
+gql`
+  query GetPaginatedTransactions($first: Int, $after: String, $before: String) {
+    me {
+      id
+      defaultAccount {
+        transactions(first: $first, after: $after, before: $before) {
+          edges {
+            cursor
+            node {
+              createdAt
+              direction
+              id
+              memo
+              settlementAmount
+              settlementCurrency
+              settlementDisplayAmount
+              settlementDisplayCurrency
+              settlementDisplayFee
+              settlementFee
+              settlementVia {
+                ... on SettlementViaIntraLedger {
+                  counterPartyUsername
+                  counterPartyWalletId
+                }
+                ... on SettlementViaLn {
+                  paymentSecret
+                  preImage
+                }
+                ... on SettlementViaOnChain {
+                  transactionHash
+                  vout
+                }
+              }
+              status
+              settlementPrice {
+                base
+                currencyUnit
+                formattedAmount
+                offset
+              }
+              initiationVia {
+                ... on InitiationViaIntraLedger {
+                  counterPartyUsername
+                  counterPartyWalletId
+                }
+                ... on InitiationViaOnChain {
+                  address
+                }
+                ... on InitiationViaLn {
+                  paymentHash
+                }
+              }
+            }
+          }
+          pageInfo {
+            endCursor
+            hasNextPage
+            hasPreviousPage
+            startCursor
+          }
+        }
+      }
+    }
+  }
+`;
+
+gql`
```

Comment:
```suggestion
```

Reply:
you don't need to close the `gql`. you can have several queries in a single `gql`

Reply:
you still don't need to close / reopen the gql bracket